testgrid/config/config.yaml

```diff
@@ -2863,8 +2863,62 @@ dashboards:
 
 - name: sig-apps
   dashboard_tab:
-  - name: TODO
-    test_group_name: ci-kubernetes-e2e-gce-slow  # TODO(fejta): fix
+  - name: gce
+    test_group_name: ci-kubernetes-e2e-gce
+    base_options: 'include-filter-by-regex=sig-apps'
+    description: 'apps gce e2e tests for master branch'
+  - name: gke
+    test_group_name: ci-kubernetes-e2e-gke
+    base_options: 'include-filter-by-regex=sig-apps'
+    description: 'apps gke e2e tests for master branch'
+  - name: gce-1.7
+    test_group_name: ci-kubernetes-e2e-gce-release-1-7
+    base_options: 'include-filter-by-regex=sig-apps'
```

Comment:
got it. it will only work on master for now as we just prepend `sig-apps` label to the test names.